specification/logic/resource-manager/Microsoft.Logic/stable/2019-05-01/logic.json

```diff
@@ -5114,459 +5125,2029 @@
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "apiName",
+            "description": "The api name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+			      "schema": {
+              "$ref": "#/definitions/ManagedApi"
+            }
+          },
+          "201": {
+            "description": "Created",
+			      "schema": {
+              "$ref": "#/definitions/ManagedApi"
+            }
+          },
+          "default": {
+            "description": "Logic error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "integrationServiceEnvironmentManagedApi"
+        ],
+        "operationId": "IntegrationServiceEnvironment_ManagedApi",
+        "description": "Deletes the integration service environment managed Api.",
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Deletes the integration service environment managed Apis": {
+          "$ref": "./examples/IntegrationServiceEnvironments_ManagedApis_Delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "name": "resourceGroupName",
+            "description": "The resource group name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "integrationServiceEnvironmentName",
+            "description": "The integration service environment name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "apiName",
+            "description": "The api name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted"
+          },
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "Logic error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.Logic/integrationServiceEnvironments/{integrationServiceEnvironmentName}/managedApis/{apiName}/operationStatuses/{operationId}" : {
```

Comment:
API to track LRO need not be modeled in swagger, this is picked from header.

Reply:
Okay removed.